sdk/ai/azure-ai-agents/samples/agents_tools/sample_agents_browser_automation.py

```diff
@@ -0,0 +1,130 @@
+# pylint: disable=line-too-long,useless-suppression
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+"""
+DESCRIPTION:
+    This sample demonstrates how to use agent operations with the Browser Automation tool from
+    the Azure Agents service using a synchronous client.
+
+USAGE:
+    python sample_agents_browser_automation.py
+
+    Before running the sample:
+
+    pip install azure-ai-projects azure-ai-agents azure-identity
+
+    Set these environment variables with your own values:
+    1) PROJECT_ENDPOINT - The Azure AI Project endpoint, as found in the Overview
+       page of your Azure AI Foundry portal.
+    2) MODEL_DEPLOYMENT_NAME - The deployment name of the AI model, as found under the "Name" column in
+       the "Models + endpoints" tab in your Azure AI Foundry project.
+    3) AZURE_PLAYWRIGHT_CONNECTION_ID - The ID of the Azure Playwright Workspaces connection, in the format of:
+       /subscriptions/{subscription-id}/resourceGroups/{resource-group-name}/providers/Microsoft.MachineLearningServices/workspaces/{workspace-name}/connections/{connection-name}
+       You can also get the connection ID programmatically from AIProjectClient, using the call
+       `project_client.connections.get("<playwright-connection-name>").id`.
+"""
+
+import os
+from azure.ai.projects import AIProjectClient
+from azure.ai.agents.models import (
+    MessageRole,
+    RunStepToolCallDetails,
+    BrowserAutomationTool,
+    RunStepBrowserAutomationToolCall,
+)
+from azure.identity import DefaultAzureCredential
+
+project_client = AIProjectClient(endpoint=os.environ["PROJECT_ENDPOINT"], credential=DefaultAzureCredential())
+
+# [START create_agent_with_browser_automation]
+connection_id = os.environ["AZURE_PLAYWRIGHT_CONNECTION_ID"]
+
+# Initialize Browser Automation tool and add the connection id
+browser_automation = BrowserAutomationTool(connection_id=connection_id)
+
+# Create an Agent with the Browser Automation tool and process an Agent run
+with project_client:
+    agents_client = project_client.agents
+    agent = agents_client.create_agent(
```

Comment:
For create, we could also add following comment (this relates to other comment regarding delete)

        # Create a new agent that has the Browser automation tool attached.
        # NOTE: To add Browser automation tool to an existing agent, fetch it with `get_agent(agent_id)` and then,
        # update the agent with the tool.